include/nccl_ofi_mr.h

```diff
@@ -101,13 +106,22 @@ static inline uintptr_t nccl_ofi_mr_ckey_len(nccl_ofi_mr_ckey_ref ckey)
 #if HAVE_DECL_FI_MR_DMABUF
 static inline nccl_ofi_mr_ckey_t nccl_ofi_mr_ckey_mk_dmabuf(int fd, uint64_t offset, size_t len, void *base_addr)
 {
+	/* Use page aligned address to register mr. */
```

Comment:
lines 109-117 are identical with lines 133 - 139, except for the function in the print.  This begs for a helper function.

Reply:
Consolidated the logic into an impl function for fewer call parameters for now.